gpu4pyscf/dft/libxc.py

```diff
@@ -18,12 +18,28 @@
 import os
 import numpy as np
 import ctypes
+import ctypes.util
 import cupy
 from pyscf import dft
 from gpu4pyscf.dft.libxc_structs import xc_func_type
 
-_libxc = np.ctypeslib.load_library(
-    'libxc', os.path.abspath(os.path.join(__file__, '..', '..', 'lib', 'deps', 'lib')))
+import site
+path_list = [os.path.abspath(os.path.join(__file__, '..', '..', '..'))] + site.getsitepackages()
+
+for path in path_list:
+    try:
+        _libxc = np.ctypeslib.load_library(
+            'libxc', os.path.abspath(os.path.join(path, 'gpu4pyscf', 'lib', 'deps', 'lib')))
+        print(f'find libxc in {path}')
+    except Exception:
+        _libxc = None
+    if _libxc is not None:
+        break
+
+if _libxc is None:
```

Comment:
Shall we remove `import libxc` in `dft/__init__.py`  . Then the other module can work without installing or compiling libxc.

Reply:
OK. Let me try if it breaks anything.

Reply:
`libxc` is imported by `numint`. We have to remove `rks` from `dft/__init__.py` for skipping `libxc`. I change the error to a warning. Other modules still can work without libxc.